### PR TITLE
{bio}[GCC/12.2.0] Cluster-Buster v20200507

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/Cluster-Buster/Cluster-Buster-0.0-GCC-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'Cluster-Buster'
+version = '0.0'
```
Comment (branfosj) [inline easybuild/easyconfigs/c/Cluster-Buster/Cluster-Buster-0.0-GCC-12.2.0.eb:4]: Please use the date of the commit for the version. So `ac1d33cffff0c276216450ebced471595cf01488` is `20200507` (May 7, 2020).

File: easybuild/easyconfigs/c/Cluster-Buster/Cluster-Buster-0.0-GCC-12.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+sources = [{
+    'git_config': {
+        'url': 'https://github.com/weng-lab',
+        'repo_name': '%(namelower)s',
+        'commit': 'ac1d33cffff0c276216450ebced471595cf01488',
+    },
+    'filename': SOURCE_TAR_GZ,
+}]
+checksums = ['1654f16b0da0203268a2c785d13abeac24aa6e4dfacb4351a8e79905c585c277']
```
Comment (branfosj) [inline easybuild/easyconfigs/c/Cluster-Buster/Cluster-Buster-0.0-GCC-12.2.0.eb:20]:
```suggestion
checksums = [None]
```

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/Cluster-Buster/Cluster-Buster-0.0-GCC-12.2.0.eb (hunk 2 above)
Comment (fizwit) [reply easybuild/easyconfigs/c/Cluster-Buster/Cluster-Buster-0.0-GCC-12.2.0.eb:20]: updated-pr

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0104u25b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/1a26edd61483d9e29d84ccf64eca06ca for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @fizwit!

**Pull request merged**